docs/7.migration/11.module-authors.md

```diff
@@ -2,9 +2,9 @@
 
 ## Module Compatibility
 
-Nuxt 3 has a basic backward compatibility layer for Nuxt 2 modules using `@nuxt/kit` auto wrappers. But there are usually steps to follow to make modules compatible with Nuxt 3 and sometimes, using Nuxt Bridge is required for cross-version compatibility.
+Nuxt 3 has a basic backward compatibility layer for Nuxt 2 modules using `@nuxt/kit` auto wrappers. But there are usually steps to follow to make modules compatible with Nuxt 3, and sometimes, using Nuxt Bridge is required for cross-version compatibility.
 
-We have prepared a [Dedicated Guide](/docs/guide/going-further/modules) for authoring Nuxt 3 ready modules using `@nuxt/kit`. Currently best migration path is to follow it and rewrite your modules. Rest of this guide includes preparation steps if you prefer to avoid a full rewrite yet making modules compatible with Nuxt 3.
+We have prepared a [Dedicated Guide](/docs/guide/going-further/modules) for authoring Nuxt 3 ready modules using `@nuxt/kit`. Currently, the best migration path is to follow it and rewrite your modules. The rest of this guide includes preparation steps if you prefer to avoid a rewrite yet make modules compatible with Nuxt 3.
```

Comment:
```suggestion
We have prepared a [dedicated guide](/docs/guide/going-further/modules) for authoring Nuxt 3 ready modules using `@nuxt/kit`. Currently, the best migration path is to follow it and rewrite your modules. The rest of this guide includes preparation steps if you prefer to avoid a rewrite yet make modules compatible with Nuxt 3.
```